pkg/graph/schema.graphql

```diff
@@ -2253,6 +2274,9 @@ type Mutation {
   updateAccessibilityRequestCedarSystem(
     input: UpdateAccessibilityRequestCedarSystemInput
   ): UpdateAccessibilityRequestCedarSystemPayload
+  createSystemIntakeActionRequestEdits(
+    input: SystemIntakeRequestEditsInput!
+  ): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
```

Comment:
I could go either way on using `UpdateSystemIntakePayload` vs. just returning a `SystemIntake` object. The `UpdateSystemIntakePayload` type was something that existed before we joined, and I'm not sure it really provides a ton of value when GQL/Apollo already support returning data and errors simultaneously.

Probably not a big deal either way, but dealing with the `SystemIntake` type directly can often mean one less extra type to wrap things in.